src/components/ModalSourcesTypeEditor.tsx

```diff
@@ -167,20 +179,27 @@ class VideoSourceEditor extends React.Component<VideoSourceEditorProps> {
       });
     }
 
+    const cornerLabels: { label: string, key: string }[] = [
```

Comment:
This looks like a duplicate of the previous one, maybe there's a way to move it to a single place and use it in both places?